tests/outputs/reference/TestPar/TestPar.par

```diff
@@ -1,14 +1,18 @@
 <?xml version="1.0" encoding="ISO-8859-1" standalone="no"?>
 <parametersSet xmlns="http://www.rte-france.com/dynawo">
-  <set id="G2">
+  <set id="10160083025018531003">
     <par name="generator_KGover" type="DOUBLE" value="1"/>
-    <par name="generator_QMax0" type="DOUBLE" value="31"/>
+    <par name="generator_QMax0" type="DOUBLE" value="41"/>
```

Comment:
Why did this change?

Reply:
In fact the two generators have been swapped in the ordering. I don't have the hand on this but apparently the xmlWriter reorder the generators based on their id. Before G2 was before G3 in the xml, with the hash, G2 became 6320417593396202500 and G3 became 10160083025018531003. Therefore G3 took the place of G2 and the generator_QMax0 values have been swapped